config/resolvers/git-resolver-config.yaml

```diff
@@ -22,9 +22,22 @@ metadata:
     app.kubernetes.io/instance: default
     app.kubernetes.io/part-of: tekton-pipelines
 data:
-  # The maximum amount of time a single git resolution may take.
+  # The maximum amount of time a single anonymous cloning resolution may take.
   fetch-timeout: "1m"
-  # The git url to fetch the remote resource from.
+  # The git url to fetch the remote resource from when using anonymous cloning.
   default-url: "https://github.com/tektoncd/catalog.git"
```

Comment:
I think I missed this earlier 🤔 Why do we default to the catalog repo ?

Reply:
I have no idea, but it was already that way when I copied it over from resolution.git. =)